### ISSUE TITLE
Fix A11y: <a> element should have child content

### ISSUE THREAD

**Issue by ondrej-tucek** [ondrej-tucek/ntc-mategra#11]
```html
<a id="eu" class="link" href="https://europa.eu/" target="_blank" alt="EU" />
^
...
```
Solution, correct code:
```html
html` <a ...>Foo</a> `;
```
see more
https://open-wc.org/docs/linting/eslint-plugin-lit-a11y/rules/heading-has-content/
https://fossies.org/linux/svelte/site/content/docs/05-accessibility-warnings.md
https://geoffrich.net/posts/svelte-a11y-limits/
